src/BenchmarksDriver/Program.cs

```diff
@@ -191,6 +199,26 @@ public static int Main(string[] args)
                     webHostValue = "Kestrel";
                 }
 
+                var aspnetCoreVersion = aspnetCoreVersionOption.Value();
+                if (string.IsNullOrEmpty(aspnetCoreVersion))
+                {
+                    aspnetCoreVersion = "2.1.0-*";
+                }
+
+                var runtimeFrameworkVersion = runtimeFrameworkVersionOption.Value();
+                if (string.IsNullOrEmpty(runtimeFrameworkVersion))
+                {
+                    runtimeFrameworkVersion = "2.0.0";
+                }
+
+                var session = sessionOption.Value();
+                if (String.IsNullOrEmpty(session)) 
+                {
+                    session = Guid.NewGuid().ToString("n");
```

Comment:
Is there any point to using `Guid.NewGuid()` for session if not specified?  Would it be better to leave it as `null`?

Reply:
I realized it's probably to correlate the different metrics for a given job.